gluonnlp/model/__init__.py

```diff
@@ -105,3 +109,65 @@ def get_model(name, dataset_name='wikitext-2', **kwargs):
                 name, '\n\t'.join(sorted(models.keys()))))
     kwargs['dataset_name'] = dataset_name
     return models[name](**kwargs)
+
+def get_cache_model(name, dataset_name='wikitext-2', window=2000,
```

Comment:
This API and get_model API were missed in API doc http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-110/40/api/model.html#gluonnlp-model